lib/styled.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+let cli   = require('..');
+
+String.prototype.toTitleCase = function() {
+  return this.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
+};
+
+String.prototype.ljust = function( width, padding ) {
+	padding = padding || " ";
+	padding = padding.substr( 0, 1 );
+	if( this.length < width )
+		return this + padding.repeat( width - this.length );
+	else
+		return this;
+};
+
+Array.prototype.max = function() {
+  return Math.max.apply(null, this);
+};
+
+Object.prototype.keys = function ()
+{
+  var keys = [];
+  for(var i in this) if (this.hasOwnProperty(i))
+  {
+    keys.push(i);
+  }
+  return keys;
+};
+
+/**
+ * styled_header logs in a consistent header style
+ *
+ * @example
+ * styled_header('MyApp') # Outputs === MyApp
+ *
+ * @param {header} header text
+ * @returns {null}
+ */
+function styled_header(header) {
+  cli.log(`=== ${header}`);
```

Comment:
oh oops. I was apparently working on this the other day:

```
diff --git a/index.js b/index.js
index 0974fdc..3353e26 100644
--- a/index.js
+++ b/index.js
@@ -16,6 +16,7 @@ exports.preauth      = require('./lib/preauth');
 exports.command      = require('./lib/command');
 exports.color        = require('chalk');
 exports.action       = require('./lib/action');
+exports.header       = require('./lib/header');
 exports.extend       = require('./lib/extend');
 exports.debug        = console.debug;
 exports.mockConsole  = console.mock;
diff --git a/lib/header.js b/lib/header.js
new file mode 100644
index 0000000..e39f1aa
--- /dev/null
+++ b/lib/header.js
@@ -0,0 +1,7 @@
+'use strict';
+
+function header (text) {
+  console.log(`=== ${text}`);
+}
+
+module.exports = header;
```

I never checked it in though. We'll go with your code.